### ISSUE TITLE
Fix Django 4.0 deprecation warning

### ISSUE THREAD

**Issue by denolehov** [django-otp/django-otp#58]
Hey, upgraded django to 3.1 and started getting the following warnings:

```
django_otp/plugins/otp_totp/admin.py:90: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
    url(r'^(?P<pk>\d+)/qrcode/$', self.admin_site.admin_view(self.qrcode_view), name='otp_totp_totpdevice_qrcode'),
```

**Comment by claudep** [comment]
That was fixed in 4ea8122c0987. Do you have the latest version?

**Comment by denolehov** [comment]
Ah, I didn't realize there was a 1.0.0 version, updating `django-otp` resolved the issue, thanks and sorry for bothering!
